src/helpers/repetitions.js

```diff
@@ -107,7 +111,7 @@ exports.list = async (query, credentials) => {
 
   for (const day of Object.keys(repetitionsGroupedByDay)) {
     for (const repetition of repetitions) {
-      addRepetitionToRelevantDay(day, repetition, repetitionsGroupedByDay);
+      repetitionsGroupedByDay = addRepetitionToRelevantDay(day, repetition, repetitionsGroupedByDay);
```

Comment:
Je me demande si ca vaut pas le coup du coup de faire une methode avec toutes les boucles for ? Parce que la on recree pas mal d'objet avec des tableaux, j'ai peur que niveau perf et memoire ce soit pas fou. Qu'en penses-tu ?